web/src/upload.js

```diff
@@ -71,7 +71,7 @@ export function get_item(key, config, file_id) {
                     .closest(".message_edit_form")
                     .find(".message_edit_save");
             case "banner_container":
-                return $(`#edit_form_${CSS.escape(config.row)} .edit_form_banners`);
+                return $(`#edit_form_${CSS.escape(config.row)} #edit_form_banners`);
```

Comment:
This change is wrong -- since we allow more than one message to be in editing mode at once (makes it easy to cut/paste text), so doing this would introduce the possibility of duplicate IDs in the DOM. That's why we had this construction to use a unique ID in the edit form in order to find the right banner container for this message.

Reply:
Absolutely, that sounds reasonable! I've made a note to myself to revert this change.